designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview-other.component.html

```diff
@@ -1,8 +1,272 @@
 <div style="background:white;">
     <!-- Project Header Start -->
     <div ng-if="!$ctrl.ui.loading">
-        <!-- pub preview metadata -->
-        <prj-pub-preview-metadata></prj-pub-preview-metadata>
```

Comment:
Are you able to fit the citation in the prj-pub-preview-metadata template with a flag for type Other projects?  might help from re-adding the copy-pasted code back into this file.